modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayEnvironmentsAdministrator.testcase

```diff
@@ -18,47 +18,93 @@ definition {
 	tearDown {
 		var testPortalInstance = PropsUtil.get("test.portal.instance");
 
+		TestrayManage.deleteCategory();
+
 		Testray.tearDownCP();
 
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
 	}
 
-	@description = "This is a test for LPS-157239 - Verify if As an Administrator I should be able to add and delete a Factor Category"
-	@ignore = "Test Stub"
+	@description = "This is a test for LPS-157327 - Verify if As an Administrator I should be able to add and delete a Factor Option"
 	@priority = "5"
-	test AdminCanAddAndDeleteFactorCategory {
+	test AdminCanAddAndDeleteFactorOption {
+		task ("Go to Categories on Manage Sidebar and Add Category") {
+			ApplicationsMenu.gotoSite(site = "Testray 2");
 
-		// TODO LPS-157239 AdminCanAddAndDeleteFactorCategory pending implementation
+			TestrayLayout.clickManageOnSidebar();
 
-	}
+			TestrayManage.selectOptionsManage(itemValue = "Categories");
 
-	@description = "This is a test for LPS-157327 - Verify if As an Administrator I should be able to add and delete a Factor Option"
-	@ignore = "Test Stub"
-	@priority = "5"
-	test AdminCanAddAndDeleteFactorOption {
+			TestrayManage.addNewCategory(category = "Environments_2 Factor Category");
+		}
+
+		task ("Go to Options on Manage Sidebar and Add New Option") {
+			TestrayLayout.clickManageOnSidebar();
+
+			TestrayManage.selectOptionsManage(itemValue = "Options");
+
+			TestrayManage.createNewOption(
+				categorie = "Environments_2 Factor Category",
+				key_value = "Environments_2 Factor Option");
+		}
+
+		task ("Delete Option, Go to Options and state if deleted") {
+			TestrayManage.deleteOption(nameOptions = "Environments_2 Factor Option");
 
-		// TODO LPS-157327 AdminCanAddAndDeleteFactorOption pending implementation
+			TestrayManage.deleteCategory();
```

Comment:
Please check the comment below the deleteCategory macro and change this test case accordingly.